5.6/alpine/Dockerfile

```diff
@@ -0,0 +1,72 @@
+FROM alpine:3.3
+
+ENV PHP_INI_DIR /usr/local/etc/php
+RUN mkdir -p $PHP_INI_DIR/conf.d
+
+ENV PHP_VERSION 5.6.18
+ENV PHP_FILENAME php-5.6.18.tar.xz
+ENV PHP_SHA256 54dd9106c3469bc7028644d72ac140af00655420bbaaf4a742a64e9ed02ec1b0
```

Comment:
They do, you can check either in imagelayer or using `docker run --rm -v /var/run/docker.sock:/var/run/docker.sock nate/dockviz images -t`